packages/console/src/pages/SignInExperience/PageContent/SignUpAndSignIn/SignUpForm/use-sign-up-password-listeners.ts

```diff
@@ -57,13 +65,13 @@ const useSignUpPasswordListeners = () => {
       }
     );
 
-    if (submitCount) {
+    if (submitCountRef.current > 0) {
```

Comment:
why do we need to check the submit count here to trigger something?

Reply:
discussed offline comments added.